<launch>

  <!-- Follower nodelet -->
  <node pkg="nodelet" type="nodelet" name="turtlebot_follower" args="load turtlebot_follower/TurtlebotFollower openni_manager" respawn="true">
    <param name="x_scale" value="7.0" />
    <param name="z_scale" value="2.0" />
    <param name="min_x" value="-0.2" />
    <param name="max_x" value="0.2" />
    <param name="min_y" value="0.1" />
    <param name="max_y" value="0.5" />
    <param name="max_z" value="1.0" />
    <param name="goal_z" value="0.6" />
  </node>	   

</launch>
